docker/Dockerfile

```diff
@@ -1,72 +1,47 @@
-# use debian as base image
-FROM debian:stretch
+# Use mambaforge as the base image
+# Builds in ~9.5 min and is ~ 2.5 GB on a windows laptop
+FROM mambaorg/micromamba:0.15.3
 
 # Label image
 LABEL \
     "Description"="Container for open source time series InSAR processing with Mintpy" \
     "Github Source"="https://github.com/insarlab/MintPy/" \
     "Installation"="https://github.com/insarlab/MintPy/blob/main/docs/installation.md" \
-    "Dockerfile Author"="Andre Theron" \
-    "Email"="andretheronsa@gmail.com"
+    "Dockerfile Author"="Forrest Williams" \
+    "Email"="forrestfwilliams@icloud.com"
 
-# Install useful programs
-RUN apt-get update && \
-    apt-get install -y \
-    git \
-    python3 \
-    python3-pip \
-    wget \ 
-    vim \
-    zip \
-    && apt-get clean \
-    && rm -rf /var/lib/apt/lists/*
+# Set user name (the micromamba image comes with the micromamba user)
+ARG user=micromamba
 
-# Change default shell from dash to bash permanently
-SHELL ["/bin/bash", "-c"]
+# Clone MintPy and PyAPS repositories
+USER root
+RUN apt-get update && \
+    apt-get install -y git vim && \
+    rm /var/lib/apt/lists/* && \
+    mkdir /home/micromamba/tools && \
+    git clone https://github.com/insarlab/MintPy.git /home/micromamba/tools/MintPy && \
+    git clone https://github.com/yunjunz/PyAPS.git /home/micromamba/tools/PyAPS
```

Comment:
I think you can switch back to the micromamba user before these and you'll avoid any potential ownership issues -- these are being owned by root, and then environment below is being created by root

Reply:
you've also hard-coded `micromamba` "user" here, but not in the ARGs below

Reply:
Thank you @jhkennedy , I see what you mean about the username, I'll go hard-code `micromamba` as the user and fix the other points you mentioned.